HTML/img/Simon Forgets/SimonForgets.svg

```diff
@@ -0,0 +1 @@
th d="M268 206.76h47.28v47.3H268z" fill="#f58231" stroke="#000" stroke-width="3.63"/><path d="M31.79 265h47.28v47.3H31.8z" fill="#ff96c2" stroke="#000" stroke-width="3.63"/><path d="M90.85 265h47.28v47.3H90.85z" stroke="#000" stroke-width="3.63"/><path d="M149.9 265h47.28v47.3H149.9z" fill="#e61932" stroke="#000" stroke-width="3.63"/><path d="M208.95 265h47.28v47.3h-47.28z" fill="#7b14b4" stroke="#000" stroke-width="3.63"/><path d="M268 265h47.28v47.3H268z" fill="#0ff" stroke="#000" stroke-width="3.63"/><g aria-label="1" style="line-height:1.25;-inkscape-font-specification:'7-Segment';text-align:center" font-weight="400" font-size="64" letter-spacing="0" word-spacing="0" font-family="7-Segment" text-anchor="middle" fill="#00ff01"><path d="M191.23 98.24h121.65v83.87H191.23z" fill="#0d0d0d"/><path d="M217.62 122.18l-4.37-4.63q1.75-1.37 3.94-1.37h12.75l1.18.12-2.5 5.88zm11.88.94l2.75-6.44q3.5 1.56 3.31 5.5l-1 15.62-2.12 1.57-3.82-3zm-1.3 20.93l4.11-3 1.94 1.57-.94 15.56q-.25 4-4 5.56l-2-6.44zm-1.83 14.13l1.75 5.94-1.18.06H214.2q-2.2 0-3.75-1.31l4.93-4.7zm-11.8-.81l-4.95 4.62q-1.25-1.62-1.06-3.81l.94-15.56 2.12-1.57 3.82 3zm1.3-21l-4.12 3-1.94-1.57 1-15.62q.13-2.13 1.56-3.75l4.38 4.62zm31.75-14.2l-4.37-4.62q1.75-1.37 3.94-1.37h12.75l1.18.12-2.5 5.88zm11.88.95l2.75-6.44q3.5 1.56 3.31 5.5l-1 15.62-2.12 1.57-3.82-3zm-1.3 20.93l4.11-3 1.94 1.57-.94 15.56q-.25 4-4 5.56l-2-6.44zm-1.83 14.13l1.75 5.94-1.18.06H244.2q-2.2 0-3.75-1.31l4.93-4.7zm-11.8-.81l-4.95 4.62q-1.25-1.62-1.06-3.81l.94-15.56 2.12-1.57 3.82 3zm1.3-21l-4.12 3-1.94-1.57 1-15.62q.13-2.13 1.56-3.75l4.38 4.62zm43.63-13.25l2.75-6.44q3.5 1.56 3.31 5.5l-1 15.62-2.12 1.57-3.82-3zm-1.3 20.93l4.11-3 1.94 1.57-.94 15.56q-.25 4-4 5.56l-2-6.44z"/></g><path d="M34.19 45.71h136.4v136.4H34.18z" fill="#646464"/><g stroke="#000" stroke-width=".39"><path d="M102.38 52.49a8 8 0 00-8 8 8 8 0 008.01 8 8 8 0 008-8 8 8 0 00-8-8z" fill="#7b14b4" stroke-width=".358956"/><path d="M70.99 62.7a8 8 0 00-8 8 8 8 0 008 8 8 8 0 008-8 8 8 0 00-8-8z" fill="#f032e6" stroke-width=".358956"/><path d="M133.78 62.7a8 8 0 00-8 8 8 8 0 008 8 8 8 0 008-8 8 8 0 00-8-8z" fill="#fff" stroke-width=".358956"/><path d="M51.6 89.4a8 8 0 00-8.01 8.01 8 8 0 008 8 8 8 0 008.01-8 8 8 0 00-8-8z" fill="#4363d8" stroke-width=".358956"/><path d="M153.18 89.4a8 8 0 00-8.01 8.01 8 8 0 008 8 8 8 0 008.02-8 8 8 0 00-8.01-8z" fill="#ff96c2" stroke-width=".358956"/><path d="M51.6 122.41a8 8 0 00-8.01 8 8 8 0 008 8 8 8 0 008.01-8 8 8 0 00-8-8z" fill="#0ff" stroke-width=".358956"/><path d="M153.18 122.41a8 8 0 00-8.01 8 8 8 0 008 8 8 8 0 008.02-8 8 8 0 00-8.01-8z" fill="#e61932" stroke-width=".358956"/><path d="M70.99 149.12a8 8 0 00-8 8 8 8 0 008 8 8 8 0 008-8 8 8 0 00-8-8z" fill="#3cb44b" stroke-width=".358956"/><path d="M133.78 149.12a8 8 0 00-8 8 8 8 0 008 8 8 8 0 008-8 8 8 0 00-8-8z" fill="#f58231" stroke-width=".358956"/><path d="M102.38 159.34a8 8 0 00-8 8 8 8 0 008.01 8 8 8 0 008-8.01 8 8 0 00-8-8z" fill="#ffe119" stroke-width=".358956"/></g></svg>
```

Comment:
component svgs normally go in img/Component/<Module Name>.svg
the module name also usually has a space in it